src/NorthwindWebsite/Business/Services/Implementations/CategoryService.cs

```diff
@@ -22,4 +28,31 @@ public CategoryService(ICategoryRepository categoryRepository)
 
         return categories.ToDictionary(c => c.CategoryId, c => c.CategoryName);
     }
+
+    public async Task<FileUploadDto> GetFileUploadModel(int id)
+    {
+        var fileUploadModel = new FileUploadDto
+        {
+            CategoryId = id,
+            MaximumFileSize = _appSettings.FileUploadOptions.ImageMaxSize
+        };
+
+        return await Task.FromResult(fileUploadModel);
```

Comment:
Maybe I miss something, but why do you return just model and this method has `Task` signature? If it is a simple mapper - make it's name appropriate, please.

Reply:
@dotnet-fizzyy  it is mapped to a Task in the service mock.